### ISSUE TITLE
Add host identifying information to Session Command logs

### ISSUE THREAD

**Issue by corkrean** [gravitational/teleport#16593]
**What would you like Teleport to do?**
Add information that makes it possible to associate a Session Command log with a host. Useful information could include server_labels, and server_hostname . 

**What problem does this solve?**
Presently, users have to reference the sid from the Session Command log to the Session Started log in order to determine what host the command was executed on. This is cumbersome. 

**If a workaround exists, please include it.**
When you've identified a Session Command log you'd like to investigate further, you will need to query your SIEM to find the Session Start log that contains the corresponding sid in order to find what host the command was run on.

**Comment by russjones** [comment]
@vitorenesduarte Take a look at the `session.end` event which has all of this information in it. Essentially what we want is to add that information into the `session.command` events which are created when you do a SSH exec. An example would be running the following command:

```
$ tsh ssh server01 date
```

**Comment by vitorenesduarte** [comment]
I'm seeing the following 4 audit events being generated if I run e.g. `tsh ssh ubuntu@ip-172-31-11-148 date` (in my cloud tenant):

<img width="957" alt="Screenshot 2022-11-24 at 19 18 45" src="https://user-images.githubusercontent.com/4520516/203853180-2e5fbf9e-40f4-49be-856d-58ccfe32499c.png">

Below are the contents of __Command Execution__ (i.e. `exec`) and __Session Ended__ (i.e. `session.end`):

```json
{
  "addr.local": "...",
  "addr.remote": "...",
  "cluster_name": "vitor.teleport.sh",
  "code": "T3002I",
  "command": "date",
  "ei": 0,
  "event": "exec",
  "exitCode": "0",
  "login": "ubuntu",
  "namespace": "default",
  "server_hostname": "ip-172-31-11-148",
  "server_id": "...",
  "sid": "d9d73238-60fe-4b68-86f0-e796f5f9fd08",
  "time": "...",
  "uid": "...",
  "user": "vitor-local"
}
```

```json
{
  "cluster_name": "vitor.teleport.sh",
  "code": "T2004I",
  "ei": 1,
  "enhanced_recording": false,
  "event": "session.end",
  "interactive": false,
  "login": "ubuntu",
  "namespace": "default",
  "participants": [
    "vitor-local"
  ],
  "server_hostname": "ip-172-31-11-148",
  "server_id": "...",
  "server_labels": {
    "teleport.internal/resource-id": "e390734e-0d66-441a-bb2c-72494d07b3ea"
  },
  "session_recording": "node",
  "session_start": "...",
  "session_stop": "...",
  "sid": "d9d73238-60fe-4b68-86f0-e796f5f9fd08",
  "time": "...",
  "uid": "...",
  "user": "vitor-local"
}
```

Instead of `session.command`, I'm only seeing `exec`.
Moreover, `exec` has `server_hostname` just like `session.end` (`exec` doesn't have `server_labels` however).

@corkrean @russjones Am I missing something? Maybe this was fixed by #13156?

**Comment by zmb3** [comment]
Sorry @vitorenesduarte, we got our wires crossed a bit. This issue refers to the `session.command` event, which part of enhanced session recording, not non-interactive exec commands w/ `tsh ssh`.
